_posts/2017-04-04-five-reasons-to-submit-tutorial.md

```diff
@@ -0,0 +1,19 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2017-04-04 08:15:00
+layout: post
+title: "5 Reasons to Submit a Tutorial Proposal to DjangoCon US"
+---
+
+Our [Call for Proposals (CFP)](https://www.papercall.io/djangocon-us-2017) deadline is quickly approaching! You have until 11:59 PM on April 10 [Anywhere on Earth](https://time.is/Anywhere_on_Earth) to submit a talk or tutorial proposal. Tutorials don’t have to be specifically about Django, and since the conference isn’t until August, the content doesn’t need to be 100% fleshed out when you submit.
+
+We’d like you to consider submitting a tutorial proposal for the following reasons:
+
+ 1. Due to the amount of preparation required, tutorials are compensated!
+ 2. You can co-teach with a friend or colleague (the fee will be split between the instructors)
+ 3. Teaching something to others will increase your own understanding of the subject
+ 4. You’ll be contributing to the Django community
+ 5. It will look very fancy on your resume  :star: :scroll:
+
+If you have any more questions about speaking at DjangoCon US, please check out our [Speaker Resources page](/../../speaking/speaker-resources/).
```

Comment:
Please update the `/../../speaking/speaker-resources/` to just be `/speaking/speaker-resources/` and it should make the Github check happy. The rest looks good! 🎉